### PR TITLE
Fix types for veracode stuff

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/routes/components.ts [hunk 1]
```diff
@@ -19,6 +19,39 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
...
+      component.attributes.veracode_results_summary?.severity?.forEach(severity => {
+        severity.category.forEach(category => {
+          severityLevels[category.severity] += category.count
+        })
+      })
+
+      return {
+        name: component.attributes.name,
+        hasVeracode,
+        pass: component.attributes.veracode_policy_rules_status === 'Pass',
```
Comment (andrewrlee) [inline server/routes/components.ts:46]: is this one mapped from the compliance status?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: server/routes/components.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/routes/components.ts:46]: Yep, I also discovered the way that types file gets generated means it stops at a particular level so I ended up having to amend it to give the entire severity bit as it just puts unknown :/

Comment (mattops) [reply server/routes/components.ts:46]: should be mapped from policy_rules_status

**Pull request merged**